DESCRIPTION

```diff
@@ -35,6 +35,7 @@ Imports:
     dplyr,
     ggplot2,
     glue,
+    gt,
```

Comment:
@lauramaxwell just did it in here since this is the PR that removes the kableExtra usage too